src/main/java/com/neva/felix/webconsole/plugins/search/core/OsgiExplorer.java

```diff
@@ -61,28 +62,50 @@ public OsgiExplorer(BundleContext context) {
 	}
 
 	public File findDir(Long bundleId) {
-		return new File(String.format(BUNDLE_PATH_FORMAT, context.getProperty(BUNDLE_STORAGE_PROP), bundleId));
+		String bundleStorage = context.getProperty(BUNDLE_STORAGE_PROP);
```

Comment:
I checked that not only empty property on clean felix is a problem.
I will put just "./felix-cache" as default value if prop not specified. One problem solved.
But there is some class loading issue when decompile servlet is called.

```
11:36:36.300 [qtp1456240109-30] WARN org.eclipse.jetty.servlet.ServletHandler - Error for /system/console/search/class-decompile
java.lang.NoClassDefFoundError: Could not initialize class com.strobel.assembler.metadata.MetadataSystem
        at com.strobel.decompiler.Decompiler.decompile(Decompiler.java:34)
        at com.neva.felix.webconsole.plugins.search.core.OsgiExplorer.decompileClass(OsgiExplorer.java:130)
        at com.neva.felix.webconsole.plugins.search.rest.ClassDecompileServlet.doGet(ClassDecompileServlet.java:64)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:618)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
        at org.apache.felix.http.base.internal.handler.ServletHandler.hand
```

To be investigated...